hooks/shotgun_launch_shotgun_app.py

```diff
@@ -9,14 +9,14 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 """
-Hook for launching the app for a publish.
+Hook for launching the Shotgun app with its engine for a published file.
```

Comment:
Not crazy about the use of "Shotgun app". It's not really a Shotgun app. It's a DCC using a Shotgun Launcher instance so it's sorta misleading. Maybe not a big deal.

Reply:
I am a little concerned about renaming this hook file. If studios have this implemented already then this could create some confusion.